msi/GitAdditionalOptionsDlg.wxs

```diff
@@ -6,7 +6,7 @@
         <Control Type="Text" Id="EnableFsCacheText" Width="275" Height="30" X="38" Y="72" TabSkip="no" Text="!(loc.GitAdditionalOptionsDlg_EnableFSCacheText)" />
         <Control Type="CheckBox" Id="EnableFsCache" Width="324" Height="15" X="20" Y="53" Integer="yes" CheckBoxValue="1" Property="UI_ENABLEFSCACHE" Text="!(loc.GitAdditionalOptionsDlg_EnableFsCacheCheckBox)" />
         <Control Type="Text" Id="EnableGcmText" Width="275" Height="40" X="38" Y="135" TabSkip="no" Text="!(loc.GitAdditionalOptionsDlg_EnableGcmText)" />
-        <Control Type="CheckBox" Id="EnableGCM" Width="324" Height="15" X="20" Y="116" CheckBoxValue="1" Integer="yes" Property="ENABLEGCM" Text="Enable Git Credential Manager for Windows">
```

Comment:
Sorry to anchor the comment here, but I cannot comment directly on the commit message: `s/Credetntials/Credentials/`